Card/card.css

```diff
@@ -0,0 +1,306 @@
+body {
+  font-family: "Quicksand", sans-serif;
```

Comment:
As you are using this font at multiple places you store it in variable.

Reply:
changes done